README.md

```diff
@@ -84,23 +78,22 @@ Tibbles have a refined print method that shows only the first 10 rows, and all t
 ``` r
 library(nycflights13)
 flights
-#> Source: local data frame [336,776 x 16]
-#> 
-#>     year month   day dep_time dep_delay arr_time arr_delay carrier tailnum
-#>    <int> <int> <int>    <int>     <dbl>    <int>     <dbl>   <chr>   <chr>
-#> 1   2013     1     1      517         2      830        11      UA  N14228
-#> 2   2013     1     1      533         4      850        20      UA  N24211
-#> 3   2013     1     1      542         2      923        33      AA  N619AA
-#> 4   2013     1     1      544        -1     1004       -18      B6  N804JB
-#> 5   2013     1     1      554        -6      812       -25      DL  N668DN
-#> 6   2013     1     1      554        -4      740        12      UA  N39463
-#> 7   2013     1     1      555        -5      913        19      B6  N516JB
-#> 8   2013     1     1      557        -3      709       -14      EV  N829AS
-#> 9   2013     1     1      557        -3      838        -8      B6  N593JB
-#> 10  2013     1     1      558        -2      753         8      AA  N3ALAA
-#> ..   ...   ...   ...      ...       ...      ...       ...     ...     ...
-#> Variables not shown: flight <int>, origin <chr>, dest <chr>, air_time
-#>   <dbl>, distance <dbl>, hour <dbl>, minute <dbl>.
+#>     year month   day dep_time sched_dep_time dep_delay arr_time
+#>    <int> <int> <int>    <int>          <int>     <dbl>    <int>
+#> 1   2013     1     1      517            515         2      830
+#> 2   2013     1     1      533            529         4      850
+#> 3   2013     1     1      542            540         2      923
+#> 4   2013     1     1      544            545        -1     1004
+#> 5   2013     1     1      554            600        -6      812
+#> 6   2013     1     1      554            558        -4      740
+#> 7   2013     1     1      555            600        -5      913
+#> 8   2013     1     1      557            600        -3      709
+#> 9   2013     1     1      557            600        -3      838
+#> 10  2013     1     1      558            600        -2      753
+#> ... with 336,766 more rows
+#> ... and 12 more variables (sched_arr_time <int>, arr_delay <dbl>, carrier
+#>   <chr>, flight <int>, tailnum <chr>, origin <chr>, dest <chr>, air_time
```

Comment:
It would be nice if we could keep the type with the variable name, but that's probably going to be tricky